### PR TITLE
improve(relayer): Ignore deposits for disabled destination chains

### PR DESCRIPTION
**Pull request by pxrl**
Failing to do this can produce a lot of log spam that is ultimately unnecessary.

### CODE REVIEW

File: src/relayer/index.ts
Comment (pxrl) [inline src/relayer/index.ts:1]: This is an optimistic driveby change.

**Review by nicholaspai — APPROVED** [review]
